crates/sui-graphql-rpc/src/types/event.rs

```diff
@@ -17,6 +17,13 @@ use super::{
     move_value::MoveValue, sui_address::SuiAddress,
 };
 
+/// A Sui node emits one of the following events:
+/// Move event
+/// Publish event
+/// Transfer object event
+/// Delete object event
+/// New object event
```

Comment:
we might be better off having a link to the possible event variations?

Reply:
This comment does not seem right to me -- I'm not sure that publish, transfer, delete, creation create events anymore (they used to, but we stopped doing that shortly before mainnet launch, because the volume of events was too great), how about this instead:

> Move code can emit events to signal when something happens on-chain that might be relevant to off-chain services.  Off-chain services keep track of on-chain activity by subscribing to events, and/or filtering and indexing transactions that emitted specific events.